docs/source/admin-guide/install.rst

```diff
@@ -2,31 +2,139 @@
+  * LLVM split into runtime libraries (package called ``llvm``) and compile time
+    libraries/headers etc this contains a build of LLVM (package called
+    ``llvmdev``)
+
+At compile time the ``llvmdev`` and ``llvm`` packages are used to build llvmlite and
+llvmlite's dynamic libraries are dynamically linked against the libraries in the
+``llvm`` meta-package. This means at runtime ``llvmlite`` depends on the ``llvm``
+package which has the LLVM shared libraries in it (it's actually a package
+called ``libllvm`` that contains the DSOs, but the ``llvm`` package is referred to
+so as to get the ``run_exports``).
+
+Using `pip`
+-----------
+
+The Numba maintainers ship binary wheels:
+
+  * Numba wheels (``x86*`` architectures)
+  * llvmlite wheels (``x86*`` architectures)
+
+Note that the llvmlite wheels are statically linked against LLVM, as per the
+conda packages on the Numba channel. This mitigates the need for a LLVM based
+binary wheel. Not also that this, as of version 0.36 does not include the
+``aarch64`` architectures, for example installation on a RaspberryPis is not
+supported.
+
+The Numba maintainers ship an ``sdist`` for:
+
+  * Numba
+  * llvmlite
+
+Note that there is no ``sdist`` provided for LLVM. If you try and build ``llvmlite``
+from ``sdist`` you will need to bootstrap the package with your own appropriate
+LLVM.
+
+How this ends up being a problem.
+
+1. If you are on an unsupported architecture (i.e. not ``x86*``) or unsupported
+   Python version for binary wheels (e.g. Python alphas) then ``pip`` will try and
+   build Numba from ``sdist`` which in turn will try and build ``llvmlite`` from
+   ``sdist``. This will inevitably fail as the ``llvmlite`` source distribution
+   needs an appropriate LLVM installation to build.
+2. If you are using ``pip < 19.0`` then ``manylinux2010`` wheels will not
+   install and you end up in the situation in 1. i.e. something unsupported so
+   building from ``sdist``.
+
+Historically, this issues has manifested itself as the following error
+message, which included here verbatim for future reference::
+
+    FileNotFoundError: [Errno 2] No such file or directory: 'llvm-config'
+
+Things to "fix" it...
+
+1. If you are using ``pip < 19.0`` and on ``x86*``, then update it if you can, this will
+   let you use the ``manylinux2010`` binary wheels.
+
+2. If you are on an unsupported architecture, for example RaspberryPis, please
+   use ``conda`` if you have that available.
+
+3. Otherwise: you will probably need to build from source, this means providing
+   an LLVM. If you have conda available you could use this to bootstrap the
+   installation with a working ``llvm``/`llvmdev` package. If you have a system
+   LLVM that's of an appropriate version you could use that. See
+   https://llvmlite.readthedocs.io/en/latest/admin-guide/install.html#building-manually
+   and in particular note the use of the ``LLVM_CONFIG`` environment variable
+   for specifying where your LLVM install is.
+
```

Comment:
This section and the next `What not to do...` section seem to conflict? This section suggests that if you have a system LLVM that's appropriate you could use it, but the next suggests to not use a package managed one (i.e. system). Perhaps the next second should be called "What to be aware of if using a system provided LLVM" and note that a) might not work b) unlikely patches in a way all the tests will pass c) may be harder to get help.

Reply:
OK, I have reworded the section, please take another look.

Reply:
Great, thanks.